pkg/tekton/metapipeline/metapipeline.go

```diff
@@ -81,7 +81,7 @@ func CreateMetaPipelineCRDs(params CRDCreationParameters) (*tekton.CRDWrapper, e
 		revision = params.PullRef.BaseBranch
 	}
 	resources := []*pipelineapi.PipelineResource{tekton.GenerateSourceRepoResource(params.ResourceName, &params.GitInfo, revision)}
-	run := tekton.CreatePipelineRun(resources, pipeline.Name, pipeline.APIVersion, labels, params.ServiceAccount, nil, nil)
+	run := tekton.CreatePipelineRun(resources, pipeline.Name, pipeline.APIVersion, labels, params.ServiceAccount, nil, nil, nil)
```

Comment:
So here we don't need the affinity? Would it not make sense here as well?

Reply:
Would the aim not also be to separate between say build nodes and app nodes?

Reply:
I guess issue #5002 provides the answer. This is just about distributing parallel stages. For meta pipeline generation that makes no sense I guess.